add example with page id (#408)
TYPO3-Documentation/TYPO3CMS-Reference-Typoscript: Documentation/Conditions/Index.rst

      [getTSFE().type == 98]

>    Current page id equals to 17. :ref:`PAGE`::

Reviewer: Linking to the explanation of the `PAGE` TypoScript object does not add any value here IMO.

Author: https://docs.typo3.org/m/typo3/reference-typoscript/master/en-us/Setup/Page/Index.html

Only here you have an explanation for the id of the page.

Author: Before TYPO3 10 you could write the TypoScript Condition like this:

    [globalVar = TSFE:id = 279]

So I find this solution:

    [getTSFE().id == 279]

I could not find your solution at all.

Another hint could be added in order to refer to page based conditions.

Reviewer: Exactly: https://docs.typo3.org/m/typo3/reference-typoscript/10.4/en-us/Conditions/Index.html#page

Author: I have added the local link to page instead.

>    Current page id equals to 17. However the recommended way is with :ref:`condition-page`::

Reviewer: ```suggestion
   Current page id equals to 17. However, :ref:`condition-page` should be preferred::
```
